tasks/section_1_Initial_Setup.yaml

```diff
@@ -482,6 +482,16 @@
         line: "install usb-storage /bin/true"
         state: present
         create: true
+    - name: 1.1.24 Disable USB Storage | gets kernel modules that are using usb storage
+      shell: lsmod | grep -i usb_storage
+      # If grep cannot find the pattern, it will return 1 as its exit status which will cause the task to fail, hence the ignore_errors: true
+      ignore_errors: true
+      register: kernel_modules_using_usb
+    - name: 1.1.24 Disable USB Storage | disables the kernel modules that are using usb storage
```

Comment:
As this might cause unwanted impact, maybe add a conditional flag ("disable_usb_kernel_modules") to this command's execution, with default value "false"?

Reply:
solved